test_scripts/Defects/4_6/2323_SDL_must_set_NOT_AUDIBLE_to_media_app_in_case_this_app_is_activated_during_phone_call.lua

```diff
@@ -0,0 +1,95 @@
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/2323
+--
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- 2) Media app is registered.
+-- Description:
+-- SDL must set media app to LIMITED and NOT_AUDIBLE in case this app is activated during phone call
+-- Steps to reproduce:
+-- 1) SDL receives SDL.ActivateApp(<appID_of_media_app>) request from HMI during phone call (or <appID_of_navigation_app>, <appID_of_communication_app>)
+-- Expected result:
+-- SDL must respond SDL.ActivateApp (SUCCESS) to HMI
+-- send OnHMIStatus ("HMILevel: FULL, audioStreamingState: NOT_AUDIBLE") to requested for activation app
+-- send OnHMIStatus ("HMILevel: FULL, audioStreamingState: AUDIBLE") to requested-to-activate-app after the phone call ends
+-- SDL receives OnEventChanges (PHONE_CALL, isActive=false) from HMI)
+-- Actual result:
+-- SDL sends OnHMIStatus ("HMILevel: BACKGROUND, audioStreamingState: NOT_AUDIBLE") to requested for activation app and
+-- does not send OnHMIStatus ("HMILevel: FULL, audioStreamingState: AUDIBLE") to requested-to-activate-app after the phone call ends
+-- SDL receives OnEventChanges (PHONE_CALL, isActive=false) from HMI)
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Defects/commonDefects')
+local mobile_session = require('mobile_session')
+
+--[[ Local Variables ]]
+config.application1 = {
```

Comment:
There is no need to redefine all params for application1.
It is enough to redefine:
```
config.application1.registerAppInterfaceParams.isMediaApplication = true
config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
```